sample/src/main/java/co/nimblehq/sample/ui/surveylist/SurveyListActivity.kt

```diff
@@ -0,0 +1,33 @@
+package co.nimblehq.sample.ui.surveylist
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import co.nimblehq.sample.databinding.ActivitySurveyListBinding
+
+class SurveyListActivity : AppCompatActivity() {
+
+    private lateinit var surveyListViewModel: SurveyListViewModel
+    private lateinit var binding: ActivitySurveyListBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySurveyListBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        title = "Survey List"
+
+        surveyListViewModel = ViewModelProvider(this, SurveyListModelFactory())
+            .get(SurveyListViewModel::class.java)
+
+        surveyListViewModel.surveyListResult.observe(this@SurveyListActivity, {
```

Comment:
Optional: I think , just `this` is enough

Reply:
@sleepylee Yes, `this` is enough.